nitro-protocol/contracts/libraries/NitroUtils.sol

```diff
@@ -143,6 +143,10 @@ library NitroUtils {
         assembly {
             id := chainid()
         }
+        // TODO: This is a workaround for https://github.com/filecoin-project/ref-fvm/issues/1182
+        if (id == 31415926) {
+            id = 31415;
+        }
```

Comment:
😆

Reply:
Be better to put this workaround in off-chain code, if possible.

Reply:
[It turns out I was able to do this by](https://github.com/statechannels/go-nitro/pull/993/commits/f8f0e63ab26a1a55ea0f29079ca54c4092045a4b):
- Using the incorrect chain id(31415926) in the state data so the on chain chain id check passes.
- Using the correct chain id (31415) when creating the transaction signer. Wallaby is expecting transactions signed with the correct chain id, so we'll get a signature failure if we use a signer with a chain id of 31415926.